modules/ROOT/pages/troubleshooting.adoc

```diff
@@ -379,4 +349,4 @@ To fix this problem, you must remove your application from the dropins
 folder and copy it to another directory. If you must leave it in the dropins folder, you
 must disable the application monitoring by using the following code in your `server.xml` file:
 
- `applicationMonitor dropinsEnabled="false"`
+`applicationMonitor dropinsEnabled="false"`
```

Comment:
for consistency with other examples:

\----
\<applicationMonitor dropinsEnabled="false"/>
\----